statuslistener/statuslistener_test.go

```diff
@@ -446,7 +446,10 @@ func TestConsumeStatusMessage(t *testing.T) {
 
 		sender := MockEventStreamSender{TestSuite: t, StatusMessage: testEntry.StatusMessage}
 		esp := &events.EventStreamProducer{Sender: &sender}
-		avs := AvailabilityStatusListener{EventStreamProducer: esp}
+
+		avs := AvailabilityStatusListener{EventStreamProducer: esp, healthcheck: make(chan struct{}, 10)}
```

Comment:
Yeah - a buffered channel allows async production for up to 10 "slots" like a queue. if we didn't have a buffer it would block until the other thread could pick up off of the channel. 

It theoretically could be a problem, but I doubt the sub-thread would get far behind since all it is doing is updating a timestamp, but I'll move the initial call to the `if event_type == "healthcheck"` block to make sure it never happens.

Reply:
thanks!